### ISSUE TITLE
looking for possible way to train official resnet from scratch with BN unfreezed

### ISSUE THREAD

**Issue by tabrisweapon** [tensorflow/models#5958]
I've went through the official/resnet implement. It seems that freezed batch normalization is the default setting and there is no way to train resnet from scratch and update the BN variable at the same time by simply setting a flag. Programmers have to change the official code and give the loss_filter_fn a value to make the exclude_batch_norm function out of function.  
corresponding codes are in official/resnet/resnet_run_loop.resnet_model_fn.exclude_batch_norm
If I am right, wouldn't be better to have a flag responsible for this function.

### System information
- **What is the top-level directory of the model you are using**: official/resnet
- **Have I written custom code**: no
- **OS Platform and Distribution**: Linux Ubuntu 16.04
- **TensorFlow installed from**:binary
- **TensorFlow version**:1.13
- **Bazel version (if compiling from source)**:N/A
- **CUDA/cuDNN version**: cudnn7
- **GPU model and memory**:NVIDIA 1080Ti
- **Exact command to reproduce**:N/A

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tabrisweapon** [comment]
filled out necessary info

**Comment by tabrisweapon** [comment]
> Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
> OS Platform and Distribution
> TensorFlow installed from
> TensorFlow version
> Bazel version
> CUDA/cuDNN version
> GPU model and memory
> Exact command to reproduce


filled out necessary info

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
